### PR TITLE
Remove wasm-bindgen as dependency

### PR DESCRIPTION
**Pull request by Jauler**
### Problem
Remove wasm-bindgen as dependency as it is not used anymore

### :ballot_box_with_check: Definition of Done checklist
- [x] Commit history is clean ([requirements](../blob/main/docs/git_commit_messages_requirements.md))
- [x] README.md is updated
- [x] Functionality is covered by unit or integration tests

### CODE REVIEW

**Review by mathiaspeters — COMMENTED** [review]
+1

File: Cargo.lock [hunk 1]
```diff
@@ -4342,7 +4342,6 @@ dependencies = [
  "tracing",
  "uniffi",
  "uuid",
- "wasm-bindgen",
```
Comment (packgron) [inline Cargo.lock:4345]: Looks incorrect, should be etire dep

Comment (Jauler) [reply Cargo.lock:4345]: Seems like `reqwest` has a dependency to `wasm-bindgen` which is not present in our selection of feature set for request. But it is still locked in `Cargo.lock` file.

**Review by packgron — APPROVED** [review]
+1.0

**Review by mathiaspeters — APPROVED** [review]

**Review by packgron — APPROVED** [review]

**Pull request merged**